lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -721,8 +746,10 @@ void PatchEntryPointMutate::processComputeFuncs(ShaderInputs *shaderInputs, Modu
     origFunc->eraseFromParent();
   }
 
-  if (origFuncs.size() == 1 && !isComputeWithCalls())
+  if (!isComputeWithCalls()) {
+    assert(origFuncs.size() == 1 && "More than one function are disallowed for shaders");
```

Comment:
That depends on how we detect compute libraries. I.e. it would currently fail when having multiple functions in a non-cs shader and not marking it as a compute pipeline.
Thinking about it, would this fail for vs-fs pipeline? I didn’t notice any failures though.